test/KoreBuild.FunctionalTests/Utilities/TestApp.cs

```diff
@@ -41,7 +41,7 @@ public TestApp(string templateDir, string toolsSource, string source, string wor
             return await ExecuteScript(output, "build", new string[0], commandArgs);
         }
 
-        private async Task<int> ExecuteScript(ITestOutputHelper output, string script, string[] koreBuildArgs, params string[] commandArgs)
+        private Task<int> ExecuteScript(ITestOutputHelper output, string script, string[] koreBuildArgs, params string[] commandArgs)
```

Comment:
This method is no longer asynchronous. WaitForExit is blocking.

Reply:
I recommend just making the return type `int` instead of `Task<int>`

Reply:
I know, I was using the branch to test out a change on my Ubuntu VM, I'll :up::date: when I'm proposing a fix.